.github/workflows/nightly-ci-repo.yaml

```diff
@@ -0,0 +1,11 @@
+name: Nightly CI (repo level @main)
+
+on:
+  schedule:
+    - cron: "35 5 * * *" # every day @ 5:35am UTC
```

Comment:
Can we just add this to `ci-repo.yml`?

Reply:
Splitting it up helped with keeping the CI badges accurate on tackle2-ui.

Reply:
I've moved the badges to main README